Source/Controllers/MainViewControllers/SPDatabaseDocument.m

```diff
@@ -111,6 +111,7 @@ @interface SPDatabaseDocument ()
 @property (nonatomic, strong) NSImage *textAndCommandMacwindowImage API_AVAILABLE(macos(11.0));
 @property (nonatomic, strong, readwrite) SPWindowController *parentWindowController;
 @property (assign) BOOL appIsTerminating;
+@property (nonatomic, strong) NSNotificationCenter *defaultNotificationCenter;
```

Comment:
Dealloc is not called so holding it is even worse right now... Not sure if we should do this 😭

Reply:
OK, I just hate seeing 16 `[NSNotificationCenter defaultCenter] `calls...

Reply:
reverted.